src/components/NewEntry.spec.js

```diff
@@ -0,0 +1,134 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import NewEntry from './NewEntry'
+import userEvent from '@testing-library/user-event'
+
+describe('NewEntry', () => {
+  it('has radio input fields with the expected text', () => {
+    render(<NewEntry />)
+
+    const firstInputField = screen.getByDisplayValue('weiß')
+    const secondInputField = screen.getByDisplayValue('einfarbig')
+    const thirdInputField = screen.getByDisplayValue('Baumwolle')
+    const forthInputField = screen.getByDisplayValue('Figurbetont')
```

Comment:
`Fourth` and not `forth` or? I would also recommend to choose more specific names. Maybe the naming of your arrays could help?

Reply:
Or what the inputs hold.

Reply:
Yes, a typo. Have renamed them, hopefully with a better name :)